Database.cpp

```diff
@@ -1,8 +1,44 @@
 #include "Database.hpp"
+#include <algorithm>
+
+//void searchByLastName();
+//void searchByPersonalID();
 
 void Database::printDatabase() const
 {
     for(const auto& person : data)
         std::cout << person;
     std::cout << "\n";
 }
+
+// void Database::sortBySalary();
+
+void Database::sortByLastName()
+{
+    std::sort(begin(data), end(data), [] (Person * left, Person * right)
+            {
+                return left->getLastName() < right->getLastName();
+            });
+}
+
+void Database::addPerson(Person * person)
+{
+    data.push_back(person);
+}
+
+//void Database::loadFromFile();
+//void Database::saveToFile();
+
+void Database::removeByPersonalID(const unsigned long personalID)
```

Comment:
Maybe that function can be bool. 
If personalID is found in database and it was removed, return true.
If personalID not found in database, return False.